hyperspy/axes.py

```diff
@@ -469,6 +508,30 @@ class AxesManager(t.HasTraits):
 
     def __init__(self, axes_list):
         super(AxesManager, self).__init__()
+        self.events = Events()
+        self.events.axes_changed = Event("""
```

Comment:
Sure, I will remove it, and reintroduce it in the appropriate PR where it will be used.